pages/guideList/config.js

```diff
@@ -1,17 +1,25 @@
 const guideListConfig = {
   sections: [
     {
-      name: 'Getting Started',
+      name: 'Quick start',
       guides: [
         {
-          title: 'Quickstart with Skygear',
+          title: 'Quick Start for General Apps',
           description: 'Signing up for Skygear hosting and' +
             ' create your first application using our iOS/Android/JavaScript SDK.',
           baseUrl: '/guides/get-started/',
           languages: ['ios', 'android', 'js'],
         },
+        {
+          title: 'Quick Start for Chat Apps',
```

Comment:
There is a "Skygear Chat" section below, consider putting the quick start there?
I believe "General Apps" vs "Chat Apps" is confusing to new users - I think they are not at the same level.